src/components/progress-bar/progress-bar-styles.jsx

```diff
@@ -119,10 +129,10 @@ export default theme => {
       },
     },
 
-    xs: sizeModifierFn(theme, { fontSize: 10, radius: 8 }),
-    sm: sizeModifierFn(theme, { fontSize: 11, radius: 16 }),
-    md: sizeModifierFn(theme, { fontSize: 14, radius: 20 }),
-    lg: sizeModifierFn(theme, { fontSize: 16, radius: 24 }),
+    xs: sizeModifierFn(theme, sizeDict.xs),
+    sm: sizeModifierFn(theme, sizeDict.sm),
+    md: sizeModifierFn(theme, sizeDict.md),
+    lg: sizeModifierFn(theme, sizeDict.lg),
```

Comment:
Currently these classes differ from those referenced on rows 93-96 (these are generated whereas the other ones are global)